### DISCUSSION TITLE
GLOP vs CLP Linear Solver

### DISCUSSION THREAD

**Discussion by JinseGeerts** [google/or-tools#2780] [Linear Solver questions] [answered]
Hi guys

I've got a model of which GLOP says it's infeasible (it says 'No solutions exist'), however when I run the exact same model with CLP, the model is solved in a few seconds with an optimal solution. All solver settings are on their default values. I'm using the C# wrapper to run my model.

Is this behaviour caused by any solve parameters? Is this a potential bug in GLOP? Should I add this to the Issues tracker?

Please let me know what could be happening here.

Kind regards
Jinse

**Comment by lperron** [comment]
GLOP is stricter w.r.t. numerical errors.
Can you send us the model ?

**Comment by JinseGeerts** [comment]
[Model.zip](https://github.com/google/or-tools/files/7204293/Model.zip)

This is a very tiny model. I've had this issue with multiple models though.
Thanks for the quick response.

**Comment by JinseGeerts** [comment, marked as answer]
I was just informed that the environment I was running on was still using on older version of GoogleOrTools.
I ran this model on our experimental environment which does have the latest version, and the GLOP solver now returns optimal too.

Apologies for the inconvience and I guess thanks for fixing this issue in the past :)

Kind regards!
Jinse
